Tests/LibPDF/TestPDF.cpp

```diff
@@ -13,21 +13,21 @@
 
 TEST_CASE(linearized_pdf)
 {
-    auto file = Core::MappedFile::map("/home/anon/Documents/pdf/linearized.pdf").release_value();
+    auto file = Core::MappedFile::map("linearized.pdf").release_value();
```

Comment:
Please don't do incremental changes like this. Squash, maybe?

I don't know what the best way to do this is, but maybe you could conditionally `chdir("/home/anon/Documents/pdf/");` if we're executing on Serenity? There's probably a better way still.

Reply:
There was a comment to add lagom tests as a future improvement. The commit for this "future improvement" was added to this PR. 
I usually try to avoid conditional compilation especially in cases where the code actually *can* be compiled on all targets. Any other suggestion?

Reply:
-  [x] Squashed the "install change" into the first commit.